docker-compose.yml

```diff
@@ -39,7 +39,7 @@ services:
     image: redis:3-alpine
     restart: unless-stopped
   postgres:
-    image: postgres:9.5-alpine
+    image: postgres:9.5.20-alpine
```

Comment:
if this is because of the postgress break (due to the authentication) - please state it so it will be clear why this small change introduced.

Reply:
Yeah, it is, it only blocks us from testing as far as I know(though it can break deployment as well, need to check it).